PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3947,7 +3947,8 @@ bool Notepad_plus::saveScintillaParams()
 
 bool Notepad_plus::addCurrentMacro()
 {
-	vector<MacroShortcut> & theMacros = (NppParameters::getInstance())->getMacroList();
+	NppParameters * pNppParam = NppParameters::getInstance();
+	vector<MacroShortcut> & theMacros = pNppParam->getMacroList();
 
 	int nbMacro = theMacros.size();
 

```

Comment:
addCurrentMacro has three levels of indentation. Is a drive-by fix in order?

i.e. Changing 

```
if (ms.doDialog() != -1)
{
```
<huge block of code>
```
}
return false;
```

To:

```
if (ms.doDialog() == -1)
{
    return false;
}
```